apps/reaper/lib/reaper/util.ex

```diff
@@ -5,4 +5,11 @@ defmodule Reaper.Util do
   def deep_merge(left, right), do: Map.merge(left, right, &deep_resolve/3)
   defp deep_resolve(_key, %{} = left, %{} = right), do: deep_merge(left, right)
   defp deep_resolve(_key, _left, right), do: right
+
+  def get_header_value(headers, header_name) do
+    case Enum.find(headers, fn {name, _value} -> String.downcase(name) == header_name end) do
```

Comment:
I would downcase header_name as well just to be safe.  I know this is only being used in 1 place currently with lower case value but it can prevent issues down the line.

Reply:
Done